src/app/lists/ListDataview.tsx

```diff
@@ -80,18 +81,22 @@ const listsColumnStyleOptions: ColumnStyles<ListRow> = {
     ),
 };
 
-const ListsDataView: React.FC<Props> = (props) => {
+const ListsDataView: React.FC<ListDataViewProps> = ({
+    listOfIngridients,
+    setListOfIngridients: setData,
```

Comment:
oops, seems like the same thing has happened here too. If we could just use `setListOfIngridients` throughout, that'd be fab